pages/entity/_type/_.vue

```diff
@@ -37,6 +37,10 @@
           :show-content-tier-toggle="false"
           :total-results="search.totalResults"
         />
+        <BrowseSection
```

Comment:
move this down into it's own row.

Reply:
It has its own row now and I have fixed some basic styling.